Database.py

```diff
@@ -99,7 +100,7 @@ def importDB(self, importFPath: Path = Path("")) -> None:
                     if overwrite:
                         self.employeeList.remove(dupeList[0])
                         self.employeeList.append(emp)
-
+            self.exportDB(self.initFPath, adminInfo = True, showArchivedEmployees = True)
```

Comment:
Don't force an auto save right after an import, the user might not want to depending on how the import goes.